build.sbt

```diff
@@ -1,25 +1,28 @@
 name := "sangria-play-json"
 organization := "org.sangria-graphql"
 version := "1.0.6-SNAPSHOT"
```

Comment:
It'd be great to extract this into a new `version.sbt`  file (to ease release)
I can also do that in another PR.
And I would set it to `2.0.0-SNAPSHOT` to express that we are breaking backwards compatibility, as the mima check is showing.

Reply:
regarding binary compatibility: it would be nice to start declaring return types explicitly (and this causes even more binary compatibility issues) - could you review https://github.com/pjfanning/sangria-play-json/commit/1f3f51ce1c637a33f510a993e6fe640e3d98c60f ? -- if it looks ok, I can add that commit to this PR

Reply:
yes it looks ok. Thx!